content/foundations/lifecycle.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: Component lifecycle
+---
+
+These milestones summarize the maturity lifecycle of UI components in Primer. Components are required to meet all criteria _before_ being proceeding to a given milestone.
+
+## Experimental
+
+Proof-of-concept built outside of Primer.
+
+## Alpha
+
+The component is ready for preliminary usage, with breaking changes expected.
+
+- <details><summary>The component does not have external dependencies that are not relevant to end-user experiences. </summary>This could be external libraries or dependencies on libraries in an application (like GitHub.com)</details>
+- <details><summary>The component is compatible with color modes and can adapt to different themes. </summary>The component does not reference any hard-coded static values and uses functional variables. Any temporary or app-level variables have been removed. Theme values (such as a new brand scheme) can be swapped out without touching the public API of the component.</details>
+- Basic documentation exists that includes example usage of the component.
+- Primary use cases tested and reviewed.
+- The component has 100% test coverage.
+
+## Beta
+
+The component meets most maturity criteria, and use is encouraged for most scenarios.
+
+- <details><summary>The component is used in a production application in multiple instances.</summary>Expectation is three or more instances in dotcom for Primer ViewComponents, at least one or more instances for Primer React. Usage has been audited and reviewed.</details>
```

Comment:
Will we have more detail somewhere about what "audited and reviewed" means exactly?